content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -145,32 +147,34 @@ the version.
 
 ## Webhook conversion
 
-The above example has a None converter between versions which only set `apiVersion` field
-on conversion and do not change the rest of the object. API server supports webhook conversion that
+{{< note >}}
+Webhook conversion is intorduced in kubernetes 1.13 as an alpha feature. To use this feature, the
+`CustomResourceWebhookConversion` feature get should be set. Please refer to [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+{{< /note >}}
+
+The above example has a None conversion between versions which only sets the `apiVersion` field
+on conversion and does not change the rest of the object. apiserver also supports webhook conversion that
 calls an external service in case of a conversion is required, e.g. when:
```

Comment:
Is this correct?
The API server [apiserver?] also supports webhook conversions that call an external service in case a conversion is required. For example:

Reply:
we use apiserver in other docs a lot. decided to keep the pattern in this file too.